### PR TITLE
Adding edit feature

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue

- https://github.com/code-mancers/gather-bot/issues/52 

# functionalities Added or Changed

- Added the functionality to update Standup details from the Frontend

# Checklist

- [ ] Tests
- [ ] Uploaded screenshots

# Screenshots

### CODE REVIEW

File: client/src/components/Standups/StandupList.tsx [hunk 1]
```diff
@@ -8,27 +8,32 @@ import {
...
-      </ArrayField>
-      <EditButton />
-      <ShowButton />
-    </Datagrid>
-  </List>
-);
+const StandupList = (props: ListProps) => {
+  const record = useListController();
 
+  console.log("RECORD", record);
```
Comment (SujayPrabhu96) [inline client/src/components/Standups/StandupList.tsx:17]: remove

File: server/src/services/standupScheduler.ts [hunk 2]
```diff
@@ -14,10 +14,8 @@ export const scheduleStandup = async (
 ): Promise<void> => {
   const token = slackApp.client.token;
   const client = slackApp.client;
+
   members?.forEach(async (member) => {
-    //the slack api takes an argument called channel this channel value
```
Comment (SujayPrabhu96) [inline server/src/services/standupScheduler.ts:18]: Why was this removed ?

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/services/standups.service.ts [hunk 3]
```diff
@@ -155,5 +164,48 @@ export const getStandup = async (id: string) => {
...
+};
+
+export const updateStandup = async (
+  { title, questions, members, channel, days, time }: StandupDetail,
+  id: string
+) => {
+  try {
+    await updateQuestions({ id, questions });
+
+    updateTitle({ id, title });
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:195]: Should not hit database multiple times , write in single query

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/routes/standups.router.ts [hunk 4]
```diff
@@ -21,5 +22,7 @@ router.post("/", validateStandup, (req: Request, res: Response) => {
 
 router.get("/", fetchStandupHandler);
 router.get("/:id", fetchSingleStandupHandler);
+router.put("/:id", updateStandupHandler);
+router.delete("/:id");
```
Comment (SujayPrabhu96) [inline server/src/routes/standups.router.ts:26]: remove ?

Comment (ayushdevelop) [reply server/src/routes/standups.router.ts]: okay

File: server/src/services/standupScheduler.ts (hunk 2 above)
Comment (ayushdevelop) [reply server/src/services/standupScheduler.ts:18]: oh happened accidentally

File: server/src/services/standups.service.ts [hunk 5]
```diff
@@ -155,5 +156,99 @@ export const getStandup = async (id: string) => {
...
+        id: member.slackMemberId,
+        name: member.name,
+      };
+    }),
+  };
+
+  return modifiedStandup;
+};
+
+export const updateStandup = async (
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:180]: This method needs to be revisited

**Review by SujayPrabhu96 — APPROVED** [review]
